src/components/layout/footer/styles.ts

```diff
@@ -142,3 +145,88 @@ export const LinkItem = styled.li`
     color: ${(props) => props.theme.colors.white};
   }
 `
+
+// background-image: url('${footerMail}');
+export const Newsletter = styled.section`
+  grid-area: newsletter;
+
+  background-position: top right;
+  background-repeat: no-repeat;
+  @media (min-width: ${(props) =>
+      props.theme.breakpoints.md}) and (max-width: ${(props) =>
+      props.theme.breakpoints.lg}) {
+    background-image: none;
+  }
+`
+
+export const NewsletterInfo = styled.strong`
+  max-width: 480px;
+  display: inline-block;
+  font-size: ${(props) => props.theme.fontSizes.l}px;
+  line-height: ${(props) => props.theme.lineHeights.heading};
+  margin-bottom: ${(props) => props.theme.space.lg}px;
+`
+
+export const NewsletterForm = styled.form`
+  display: grid;
+  grid-template-areas: 'input button' 'message message';
+  grid-template-rows: auto auto;
+  grid-template-columns: 316px auto;
+  grid-gap: ${(props) => props.theme.space.md}px
+    ${(props) => props.theme.space.md * 1.5}px;
+  @media (max-width: ${(props) => props.theme.breakpoints.lg}) {
+    grid-template-areas: 'input' 'button' 'message';
+    grid-template-rows: auto;
+    grid-template-columns: min(316px, 100%);
+  }
+`
+
+export const NewsletterFormControl = styled.div`
+  flex: 1;
+  grid-area: input;
+`
+
+export const NewsletterInput = styled.input`
+  --form-control-bg: #2d2d50;
+  --form-control-color: ${(props) => rgba(props.theme.colors.white, 0.8)};
+  width: 100%;
+  background: var(--form-control-bg);
+  border-width: 0;
+  border-radius: ${(props) => props.theme.borderRadius.base}px;
+  border-color: var(--form-control-bg);
+  box-sizing: border-box;
+  padding: ${(props) => props.theme.space.base * 1.5}px
+    ${(props) => props.theme.space.base * 2.5}px;
+  color: var(--form-control-color);
+  font-size: ${(props) => props.theme.fontSizes.base}px;
+  outline: 0;
+  font-family: ${(props) => props.theme.fonts.body};
+  line-height: ${(props) => props.theme.lineHeights.body};
+  ::placeholder,
+  ::-webkit-input-placeholder {
+    color: var(--form-control-color);
+  }
+  :-ms-input-placeholder {
+    color: var(--form-control-color);
+  }
+`
+
+export const NewsletterInputErrMessage = styled.output`
```

Comment:
Vzhled chybového stavu byl vytvořen dříve než styleguide pro input. Můžeš prosím kouknout do [Figmy](https://www.figma.com/file/8YdGIHXLPo8zt4ZN8KJ4Hy/C.D-Web?node-id=16%3A72) a upravit styly?

Reply:
Tak jsem rovnou vytvořil kompletní Input a InputError komponenty, přidal jsem i do storybooků.

Reply:
Jedna věc je tam nejasná:
![image](https://user-images.githubusercontent.com/1295186/104107148-8f822d80-52ba-11eb-8fc0-6efce47fe35a.png)

Každá barevná varianta má jiný left-padding (16px a 24px), je to záměr? Zatím tam mám z nějakého důvodu 20px (asi už tam bylo), tak pak ještě opravím.